packages/flutter_driver/lib/src/driver.dart

```diff
@@ -267,6 +267,12 @@ class FlutterDriver {
     return GetTextResult.fromJson(await _sendCommand(new GetText(finder))).text;
   }
 
+  /// Take a screenshot.  The image will be returned as a Base64 encoded PNG.
+  Future<Map<String, dynamic>> screenshot() async {
```

Comment:
Can we decode the image prior to returning it (e.g. using [Base64Decoder](https://api.dartlang.org/stable/1.19.1/dart-convert/Base64Decoder-class.html))? The Base64 encoding is an implementation detail of our JSON-based protocol.